.eslintrc.js

```diff
@@ -1,7 +1,15 @@
 module.exports = {
   root: true,
-  extends: ['eslint-config-digitalbazaar'],
+  extends: [
+    'eslint-config-digitalbazaar',
+    // 'eslint-config-digitalbazaar/jsdoc'
```

Comment:
Why is this disabled? This should be deleted entirely or re-enabled.

Reply:
Sure, I can remove it. (I added it in a disabled state mostly as a to-do item.)

Reply:
@gannan08 commented out line removed, opened a tracking issue at https://github.com/veres-one/did-veres-one/issues/59